### DISCUSSION TITLE
Closing all VGit windows after completing git operations

### DISCUSSION THREAD

**Discussion by bennypowers** [tanvirtin/vgit.nvim#207] [Q&A]
Say I want to stage some project hunks and make a commit. Once finished, or if I decide to abort, I'm left with 3 VGit windows for the diff, which I'll have to close one by one. What command can I issue to close all VGit windows at once?

**Comment by tanvirtin** [comment]
Executing the command `:q` on a preview should close all windows. If this isn't what's happening let me know ill log an issue, this isn't ideal behaviour.
